dev/io.openliberty.org.apache.myfaces.4.0_fat/build.gradle

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2022 IBM Corporation and others.
+ * Copyright (c) 2023 IBM Corporation and others.
```

Comment:
Shouldn't this be a range: `2022, 2023` ?

Reply:
Fixed.